.github/workflows/push.yml

```diff
@@ -35,9 +50,9 @@ jobs:
         echo ::set-output name=version::$version
```

Comment:
The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/